webserver/views/datasets.py

```diff
@@ -112,7 +112,8 @@ def evaluate(dataset_id):
             db.dataset_eval.evaluate_dataset(
                 dataset_id=ds["id"],
                 normalize=form.normalize.data,
-                filter_type=form.filter_type.data,
+                eval_location=form.evaluation_location.data,
+                filter_type=form.filter_type.data
```

Comment:
Please keep the tailing comma so that changes like these don't show up. Only one line should change here.